distributed/distributed-schema.yaml

```diff
@@ -803,44 +803,47 @@ properties:
   rmm:
     type: object
     description: |
-      Configuration options for the RAPIDS Memory Manager
+      Configuration options for the RAPIDS Memory Manager.
     properties:
       pool-size:
-        type:
-        - integer
-        - "null"
-        description:
-          The size of the memory pool in bytes
+        type: [integer, 'null']
+        description: |
+          The size of the memory pool in bytes.
   ucx:
     type: object
     description: |
-      UCX provides access to other network interconnects like Infiniband and NVLINK
+      UCX provides access to other transport methods including NVLink and InfiniBand.
     properties:
+      cuda_copy:
+        type: boolean
+        description: |
+          Set environment variables to enable CUDA transfers over UCX, even if InfiniBand and
+          NVLink are not supported or disabled.
```

Comment:
```suggestion
          Set environment variables to enable CUDA support over UCX. This may be used even if
          InfiniBand and NVLink are not supported or disabled, then transferring data over TCP.
```